features/step_definitions/user_steps.rb

```diff
@@ -13,14 +13,29 @@ def see_journey_picker?
   @see_journey_picker
 end
 
+def page_mapping(page_name)
```

Comment:
It looks good and I'm sorry to pick on this again but I think the naming is a bit inconsistent. I think the pages should be named like this:
  - `IDP sign-in` - this is used in `page_heading_text(page)` method and also "sign-in" with a hyphen is a noun, which is what makes sense
  - Either "Country Picker" or "country picker" - I find mixed case confusing. I think all lower case looks better
  - Either "Prove Identity" or "prove identity" - same as above